.github/workflows/main.yml

```diff
@@ -49,17 +51,24 @@ jobs:
           distribution: 'temurin'
           java-version: 17
 
-      - name: Get module name from tag
-        id: get-module-name
+      - name: Get namespace and module name from tag
+        id: parse-tag
         run: |
           tag=${GITHUB_REF#refs/tags/}
-          echo "result=${tag/-v*}" >> "$GITHUB_OUTPUT"
+          module=$(echo "${tag%-*-v*}")
+          namespace=$(echo "${tag/-v*}" | rev | cut -d '-' -f 1 | rev)
```

Comment:
@osipxd fyi how it work

![image](https://github.com/RedMadRobot/gears-android/assets/18189767/d952d1be-9488-4a88-bc6a-64e8233b9577)